### ISSUE TITLE
Sql Formatter doesn't handle JDBI/JDBC parameters well.

### ISSUE THREAD

**Issue by dylanbaroody** [diffplug/spotless#952]
When using JDBI, adding parameters to queries uses one of the following formats:

```
where user_id = :user_id
```
or
```
user_id in ( <user_ids> )
```

The included formatter doesn't handle either of these well:

```
WHERE user_id =:user_id
```
or
```
user_id IN ( 
    < user_ids >
)
```
In particular, the IN style statement breaks because a space is added between `<` and the word. Could a parameter to configure spaces between `(`, `<` and words be added?

**Comment by nedtwigg** [comment]
PR's welcome :)

https://github.com/diffplug/spotless/tree/main/lib/src/main/java/com/diffplug/spotless/sql

**Comment by nedtwigg** [comment]
Fixed in `plugin-gradle 5.16.0` and `plugin-maven 2.16.0`, thanks to #955.
